client/my-sites/checkout/upsell-nudge/business-plan-upgrade-upsell/new-design-202301.jsx

```diff
@@ -0,0 +1,192 @@
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+
+import './style-new-design.scss';
+
+export class BusinessPlanUpgradeUpsellNewDesign extends PureComponent {
+	render() {
+		const { receiptId, translate } = this.props;
+
+		const title = translate( 'Checkout ‹ Plan Upgrade', {
+			comment: '"Checkout" is the part of the site where a user is preparing to make a purchase.',
+		} );
+
+		return (
+			<>
+				<PageViewTracker
+					path="/checkout/:site/offer-plan-upgrade/:upgrade_item/:receipt_id"
+					title={ title }
+					properties={ { upgrade_item: 'business' } }
+				/>
+				<DocumentHead title={ title } />
+				<div className="business-plan-upgrade-upsell-new-design__header">
+					{ receiptId ? this.header() : '' }
+					<div className="business-plan-upgrade-upsell-new-design__header-title">
+						{ this.title() }
+					</div>
+				</div>
+				<div className="business-plan-upgrade-upsell-new-design__container">
+					<div className="business-plan-upgrade-upsell-new-design__body">{ this.body() }</div>
+					<div className="business-plan-upgrade-upsell-new-design__image-container">
+						{ this.discount() }
+						{ this.image() }
+					</div>
+					<div className="business-plan-upgrade-upsell-new-design__footer">{ this.footer() }</div>
+				</div>
+			</>
+		);
+	}
+
+	header() {
+		const { translate } = this.props;
+
+		return (
+			<header className="business-plan-upgrade-upsell-new-design__small-header">
+				<h1 className="business-plan-upgrade-upsell-new-design__small-header-title">
+					{ translate( 'Limited time offer' ) }
+				</h1>
+			</header>
+		);
+	}
+
+	image() {
+		return (
+			<img className="business-plan-upgrade-upsell-new-design__image" src={ upsellImage } alt="" />
+		);
+	}
+
+	title() {
+		const { translate } = this.props;
+		return (
+			<>
+				<h2 className="business-plan-upgrade-upsell-new-design__title">
+					{ translate( 'Upgrade your site to the most powerful plan ever' ) }
+				</h2>
+			</>
+		);
+	}
+
+	discount() {
+		const { translate, planRawPrice, planDiscountedRawPrice } = this.props;
+
+		const discountPercentage = Math.round( 100 - ( planDiscountedRawPrice * 100 ) / planRawPrice );
+
+		return (
+			<div className="business-plan-upgrade-upsell-new-design__discount">
+				<Badge
+					key="popular"
+					type="info"
+					className="business-plan-upgrade-upsell-new-design__discount-badge"
+				>
+					{ translate( 'Save %(discountPercentage)s%', {
+						args: { discountPercentage },
+						comment: 'The percentage value. e.g.: Save 49%',
+					} ) }
+				</Badge>
+			</div>
+		);
+	}
+
+	body() {
+		const {
+			translate,
+			planRawPrice,
+			planDiscountedRawPrice,
+			currencyCode,
+			hasSevenDayRefundPeriod,
+		} = this.props;
+		return (
+			<>
+				<div className="business-plan-upgrade-upsell-new-design__column-pane">
+					<p>{ translate( 'Unlock the power of the Business Plan and gain access to:' ) }</p>
```

Comment:
The grammar doesn't work here combined with some of the bullet points below (and related, the bullet points below don't have consistent tense -- e.g. the first is "Us**ing**" but the second is just "Upload").

My favorite is the last bullet point.  When you put it together you get "Unlock the power of the Business Plan and gain access to... gaining SFTP and database access" :smile: 

So I think the wording here needs some work.

Reply:
> It looks like the experiment is for all locales, so if you want to keep that do you have to wait for translations to finish before deploying this?

We sent them for translation here, though it sounds like we might want to make changes: https://github.com/Automattic/wp-calypso/pull/71538

> So I think the wording here needs some work.

I think we could tweak it slightly. Change:
- "Upload any WordPress themes..." to "Upload**ing** any WordPress themes" and
- "Gaining SFTP and database access" to "Setting up SFTP and database credentials."

Reply:
Sounds good! Both suggestions [were changed here](https://github.com/Automattic/wp-calypso/pull/71757/commits/97639c1bfcaf894807974a5ea039e4498e37699d). We already have 75% of strings, let's see if we have all of them at the day of release 🤞

Reply:
I'm hoping we can release to as many languages as possible since the sample size estimates are showing quite low counts!